Runtime/PlayableAssets/StreamingImageSequence/StreamingImageSequencePlayableMixer.cs

```diff
@@ -158,10 +160,37 @@ public override void ProcessFrame(Playable playable, FrameData info, object play
                 if (sprite.texture != tex) {
                     m_spriteRenderer.sprite = Sprite.Create(tex, new Rect(0.0f, 0.0f, tex.width, tex.height), new Vector2(0.5f, 0.5f), 100.0f, 2, SpriteMeshType.FullRect);
                 }
-            } else if (null!=m_meshRenderer) {
-                Material mat = m_meshRenderer.sharedMaterial;
-                mat.mainTexture = tex; 
-            } else if (null!= m_image) {
+            } else if (null!=m_meshRenderer)
+            {
+                Material mat;
+                Debug.Log(m_meshRenderer.sharedMaterial + "single material");
+                if (m_meshRenderer.sharedMaterials.Length > 1)
+                {
+                    Debug.Log(m_meshRenderer.sharedMaterial + "masterial");
+                    mat = m_meshRenderer.sharedMaterials[2];
+                }
+                else
+                {
+                    
+                    mat = m_meshRenderer.sharedMaterial;
+                }
+                //mat.mainTexture = tex; 
+                mat.SetTexture("_GraphicsMap",tex);
```

Comment:
note: this swaps the _MainTexture from builtin renderer to HDRP's _GraphicsMap texture - need a better way to handle the texture input mapping